utilities/state_converters.F90

```diff
@@ -187,20 +124,20 @@ subroutine wet_to_dry_water_vapor_run(ncol, nz, pdel, pdeldry, qv, qv_dry,  &
 
      integer,          intent(in)  :: ncol
```

Comment:
The conversion from wet to dry and dry to wet mixing ratios is general. I am curious why there are separate calls for water vapor, rain, etc. to convert from wet to dry. Could we use a generic subroutine instead?

Reply:
We currently have these separate calls for legacy reasons, i.e. we had converted Kessler before we had any general way to deal with constituents in the framework.  

However, I agree that we should have generic wet <-> dry conversion routines, and now that we have a constituents object that should be doable.  I can make an issue about it so that we remember to tackle it at some point in the future.

Reply:
I just created the issue, which can be found here: #68